setup.cfg

```diff
@@ -1,4 +1,5 @@
 [flake8]
+max-line-length: 120
```

Comment:
We had this discussion when we started the project. The number is not based on monitor size, but on human comprehension ability.